MyPlayground.playground/Contents.swift

```diff
@@ -3,3 +3,45 @@ import UIKit
 /*
  1. Create a function that takes an integer and returns a boolean which indicates if the number is prime
  2. (optional for extra bonus) Create a function that takes an integer and prints all prime numbers before this integer */
+
+func isPrime(number: Int) -> Bool {
+    if number > 1 && number < 4 {
+        return true
+    }
+    if number > 1 && number / 1 == number {
```

Comment:
`number / 1 == number` is always true 😄

Reply:
and `number > 1` is already checked at the line 8

Reply:
I can remove this check "number / 1 == number"
second if else i need to check if number is negative

Reply:
i still have to have one more if else to check if number > 1 and return FALSE if its not true